docs/topics/wildfly-swarm/docs/reference/fractions/elytron.adoc

```diff
@@ -0,0 +1,996 @@
+
+swarm.elytron.key-stores._KEY_.attribute-synchronized:: 
+The time this KeyStore was last loaded or saved. Note: Some providers may continue to apply updates after the KeyStore was loaded within the application server.
+
+swarm.elytron.key-stores._KEY_.credential-reference:: 
+The reference to credential stored in CredentialStore under defined alias or clear text password.
+
+swarm.elytron.key-stores._KEY_.loaded-provider:: 
+Information about the provider that was used for this KeyStore.
+
+swarm.elytron.key-stores._KEY_.modified:: 
+Indicates if the in-memory representation of the KeyStore has been changed since it was last loaded or stored.  Note: For some providers updates may be immediate without further load or store calls.
+
+swarm.elytron.key-stores._KEY_.path:: 
+The path to the KeyStore file.
+
+swarm.elytron.key-stores._KEY_.provider-name:: 
+The name of the provider to use to load the KeyStore, disables searching for the first Provider that can create a KeyStore of the specified type.
+
+swarm.elytron.key-stores._KEY_.providers:: 
+A reference to the providers that should be used to obtain the list of Provider instances to search, if not specified the global list of providers will be used instead.
+
+swarm.elytron.key-stores._KEY_.relative-to:: 
+The base path this store is relative to.
+
+swarm.elytron.key-stores._KEY_.required:: 
+Is the file required to exist at the time the KeyStore service starts?
+
+swarm.elytron.key-stores._KEY_.size:: 
+The number of entries in the KeyStore.
+
+swarm.elytron.key-stores._KEY_.state:: 
+The state of the underlying service that represents this KeyStore at runtime, if it is anything other than UP runtime operations will not be available.
+
+swarm.elytron.key-stores._KEY_.type:: 
+The type of the KeyStore, used when creating the new KeyStore instance.
+
+swarm.elytron.ldap-key-stores._KEY_.alias-attribute:: 
+The name of LDAP attribute, where will be item alias stored.
+
+swarm.elytron.ldap-key-stores._KEY_.certificate-attribute:: 
+The name of LDAP attribute, where will be certificate stored.
+
+swarm.elytron.ldap-key-stores._KEY_.certificate-chain-attribute:: 
+The name of LDAP attribute, where will be certificate chain stored.
+
+swarm.elytron.ldap-key-stores._KEY_.certificate-chain-encoding:: 
+The encoding of the certificate chain.
+
+swarm.elytron.ldap-key-stores._KEY_.certificate-type:: 
+The type of the Certificate.
+
+swarm.elytron.ldap-key-stores._KEY_.dir-context:: 
+The name of DirContext, which will be used to communication with LDAP server.
+
+swarm.elytron.ldap-key-stores._KEY_.filter-alias:: 
+The LDAP filter for obtaining an item of the KeyStore by alias. If this is not specified then the default value will be (alias_attribute={0}). The string '{0}' will be replaced by the searched alias and the 'alias_attribute' value will be the value of the attribute 'alias-attribute'.
```

Comment:
clean this up to make all values use backticks. Some use single quotes, some are unformatted.

Reply:
Note to self: not easily fixable in our sources.